### PR TITLE
[vault-token-injector] Allow setting vaultTokenFile

### PR DESCRIPTION
**Pull request by sudermanjr**
Blocked by the PR on vault-token-injector that creates these features. Expecting release 0.2.0

**Why This PR?**
Utilize a new feature to read the vault token from a file.

**Changes**
* Adds a new value to specify the location of the vault token file and pass that to an arg
* Add log verbosity
* Move circle token to a secret rather than just a printed env var

**Checklist:**

* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] Any new values have been added to the README for the Chart, or helm-docs has been run for the charts that support it.

### CODE REVIEW

**Review by lucasreed — COMMENTED** [review]
Wouldn't we need a configmap/secret to be created to mount as a file for this to work?

My brain is a bit tired so I could be misunderstanding.

**Comment by sudermanjr** [comment]
I should have put a description:

This is intended to work with something like vault-k8s, which injects the token as a file into the running pod.

File: incubator/vault-token-injector/templates/deployment.yaml [hunk 1]
```diff
@@ -37,14 +37,22 @@ spec:
...
+            - -v{{ .Values.logLevel }}
           volumeMounts:
           - name: config
             mountPath: /config
+          envFrom:
+          - secretRef:
+              name: {{ include "vault-token-injector.fullname" . }}
           env:
           - name: "CIRCLE_CI_TOKEN"
             value: {{ .Values.circleToken | quote }}
```
Comment (lucasreed) [inline incubator/vault-token-injector/templates/deployment.yaml]: Should we remove this since we're pulling it from a secret now?

**Review by lucasreed — COMMENTED** [review]
In-line question

File: incubator/vault-token-injector/templates/deployment.yaml (hunk 1 above)
Comment (sudermanjr) [reply incubator/vault-token-injector/templates/deployment.yaml]: Good catch

**Pull request closed without merging**